packages/editor-ui/src/components/PersonalizationModal.vue

```diff
@@ -18,59 +18,82 @@
-						<n8n-option :value="SECURITY_WORK_AREA" label="Security" />
-						<n8n-option :value="SUPPORT_WORK_AREA" label="Support" />
-						<n8n-option :value="OTHER_WORK_AREA_OPTION" label="Other (please specify)" />
-					</n8n-select>
-				</n8n-input-label>
-
-				<n8n-input
-					v-if="otherWorkAreaFieldVisible"
-					:value="values[OTHER_WORK_AREA_KEY]"
-					placeholder="Specify your work area"
-					@input="(value) => onInput(OTHER_WORK_AREA_KEY, value)"
-				/>
-
 				<n8n-input-label label="How are your coding skills?">
 					<n8n-select :value="values[CODING_SKILL_KEY]" placeholder="Select..." @change="(value) => onInput(CODING_SKILL_KEY, value)">
 						<n8n-option
-							label="0 (Never coded)"
+							label="0. Never coded"
 							value="0"
 						/>
 						<n8n-option
-							label="1"
+							label="1. I get stuck too quickly to achieve much"
 							value="1"
 						/>
 						<n8n-option
-							label="2"
+							label="2. I can code some useful things, but I spend a lot of time stuck"
 							value="2"
 						/>
 						<n8n-option
-							label="3"
+							label="3. I know enough to be dangerous, but I'm no expert"
 							value="3"
 						/>
 						<n8n-option
-							label="4"
+							label="4. I can figure most things out"
 							value="4"
 						/>
 						<n8n-option
-							label="5 (Pro coder)"
+							label="5. I can do almost anything I want, easily (pro coder)"
 							value="5"
 						/>
 					</n8n-select>
 				</n8n-input-label>
 
+				<n8n-input-label label="Which areas do you mainly work in?">
+					<n8n-select :value="values[WORK_AREA_KEY]" multiple placeholder="Select..." @change="(value) => onInput(WORK_AREA_KEY, value)">
+						<n8n-option :value="FINANCE_WORK_AREA" label="Finance" />
+						<n8n-option :value="HR_WORK_AREA" label="HR" />
+						<n8n-option :value="IT_ENGINEERING_WORK_AREA" label="IT / Engineering" />
+						<n8n-option :value="LEGAL_WORK_AREA" label="Legal" />
+						<n8n-option :value="MARKETING_WORK_AREA" label="Marketing" />
+						<n8n-option :value="OPS_WORK_AREA" label="Operations" />
+						<n8n-option :value="PRODUCT_WORK_AREA" label="Product" />
+						<n8n-option :value="SALES_BUSINESSDEV_WORK_AREA" label="Sales / Bizdev" />
+						<n8n-option :value="SECURITY_WORK_AREA" label="Security" />
+						<n8n-option :value="SUPPORT_WORK_AREA" label="Support" />
+						<n8n-option :value="EXECUTIVE_WORK_AREA" label="Executive team" />
+						<n8n-option :value="OTHER_WORK_AREA_OPTION" label="Other (please specify)" />
+						<n8n-option :value="NOT_APPLICABLE_WORK_AREA" label="I'm not using n8n for work" />
+					</n8n-select>
+				</n8n-input-label>
+				<n8n-input
+					v-if="otherWorkAreaFieldVisible"
+					:value="values[OTHER_WORK_AREA_KEY]"
+					placeholder="Specify your work area"
+					@input="(value) => onInput(OTHER_WORK_AREA_KEY, value)"
+				/>
+
+				<section v-if="showOtherQuestion">
```

Comment:
nitpick - `showOtherIndustryQuestion` or something like that might be clearer since we already have `other` work area question